### PR TITLE
[#111] Add tracking scripts to index.html

### PR DESCRIPTION
**Pull request by kamilant**
*Removed segment from website
*Added tracking scripts GoogleAnalytics/Drift/AdWords/FacebookPixel
*Resolves: #111

### CODE REVIEW

**Comment by mkucharz** [comment]
Looks good!

File: index.html [hunk 1]
```diff
@@ -18,9 +18,56 @@
...
-      }}();
+
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-82338023-1', 'auto');
```
Comment (kkotelczuk) [inline index.html]: I think keys should not be public.

File: index.html [hunk 2]
```diff
@@ -18,9 +18,56 @@
...
+      ga('send', 'pageview');
```
Comment (kkotelczuk) [inline index.html]: I think data should be sent from each site, not only in index.html.

Comment (mkucharz) [reply index.html]: Keys ar public anyway.. but we maybe should move them to Circle variable and have two sets staging/production?

File: src/utils/tracker.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return function wrapWithTracker(WrappedComponent) {
+    class TrackerComponent extends Component {
+      static propTypes = {
+        pattern: PropTypes.string,
+        location: PropTypes.object,
+      }
+
+      componentDidMount() {
+        if (!this.props.pattern) {
+          window.ga('send', 'pageview', `404${location.pathname}`);
```
Comment (kkkowalski) [inline src/utils/tracker.js:13]: 👍

**Review by kkkowalski — APPROVED** [review]
LGTM

**Pull request merged**